olm/jenkins/ci/Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+@Library('olcne-pipeline') _
+import com.oracle.olcne.pipeline.BranchPattern
+
+String version = '0.35.0'
+String imgTag  = "v" + version
+
+olcnePipeline(
+        branchPattern: new BranchPattern(master: 'oracle/release/.*', feature: '(?!^oracle/release/$)(^.*$)'),
+        containers: [('container-registry.oracle.com/olcne/ui:' + imgTag): 'olcne/ui:' + imgTag],
+        platforms: ['ol8'],
+        architectures: ['x86_64', 'aarch64'],
+        publishImage: false,
+        yumOL8Repos: ['ol8_appstream'],
+        ol8ModulesToDisable: ['go-toolset:ol8'],
+        ol8ModulesToEnable: ['nodejs:20'],
+        preBuildCommands: [
+		'podman pull container-registry.oracle.com/olcne/base:latest',
```

Comment:
you can replace this with:
container-registry.oracle.com/olcne/ock:base-image

in your Dockerfile

Reply:
could you please help to make this applies to the `oracle-template` branch as well thanks